wordpress-mobile/WordPress-iOS
# [Stats Refresh] Period Post Stats loading blocks

Refs. #12147 

This PR is the last part ( 🎉🎉🎉🎉🎉🎉) of the Load Blocks Independently, already implemented in the DWMY overview and details. It adds async loading in the Post Stats and its details.

## To test:
• Open Stats -> DWMY from a Site you never opened before (or from a fresh install)
• Open a Post Stats from the Posts and Pages cards
• You should see the Ghost rows animated
• Click on the View More button to check the details and the Ghost views
• Do the same test from Insights -> LPS and from the Posts List

Update release notes:
